### DISCUSSION TITLE
Anonymize/Hashing Sensitive Data

### DISCUSSION THREAD

**Discussion by AhmAmri** [apache/hop#4583] [Q&A] [answered]
Hi All, 

I want to ask if there is a transform that I can use to anonymize the data within my pipeline. I want to hash sensitive data like patient name, birthdate  automatically and showing them anonymized in my output.

Is that possible?

Thanks in advance :)

**Comment by bamaer** [comment, marked as answer]
There definitely are other ways to do this too, but the [Add a checksum](https://hop.apache.org//manual/latest/pipeline/transforms/addchecksum.html) transform is one option.
